web/source/tsconfig.embedded.json

```diff
@@ -6,7 +6,6 @@
   "files": [
 		"kmwbase.ts",
 		"keymanweb.ts",
-		"kmwosk.ts", 
```

Comment:
You've removed reference to kmwosk.ts but not added the new files. I know that they'll be picked up by reference but perhaps it's good to refer to them all here anyway?